style.css

```diff
@@ -0,0 +1,32 @@
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  margin: 0 auto;
+  border: 2px solid rgb(48, 18, 2);
+}
+
+h1 {
+  text-align: center;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
+
+#extra {
+  text-align: center;
+  color: rgb(27, 12, 1);
+  font-style: italic;
+}
+
+body {
+  background-color: rgb(227, 219, 212);
+}
+
+#img-caption {
+  text-align: center;
+  font-size: smaller;
+}
```

Comment:
- Your img element should have a display of block.
- Failed:Your #image should have a max-width of 100%.
- Failed:Your #image should be centered within its parent.